### ISSUE TITLE
Client page

### ISSUE THREAD

**Issue by Yuaning** [atech/postal#784]
Hi, hello, meet again. This time I would like to ask a question from the client. Our project does not have its own web client, does it?

**Comment by willpower232** [comment]
Postal has a web interface for managing it but all the messages sent through Postal should be handled via a separate system that knows to pass emails to Postal for sending. Is that what you mean?

**Comment by Yuaning** [comment]
Yes, it would be nice to have pages to accept send management and so on

**Comment by willpower232** [comment]
Unfortunately there is no interface beyond the ability to send messages and administration. If you want to add extra controls then these would be better located in your own application or a separate system.

**Comment by Yuaning** [comment]
🆗
